google/cloud/documentai_toolbox/wrappers/hocr.py

```diff
@@ -0,0 +1,241 @@
+    hocr_lines = dataclasses.field(init=False, repr=False)
+    hocr_lines: List[hOCR_Line]
+    bounding_box = dataclasses.field(init=False, repr=False)
+    bounding_box: str
+    id = dataclasses.field(init=False, repr=False)
+    id: str
+
+    def __post_init__(self):
+        self.hocr_lines = _load_hocr_lines(lines=(self.lines), page=(self.page), document_text=(self.document_text))
+        self.bounding_box = _get_bounding_box(object=(self.documentai_paragraph), dimensions=(self.dimensions))
+
+
+@dataclasses.dataclass
+class hOCR_Block:
+    page = dataclasses.field(repr=False)
+    page: documentai.Document.Page
+    paragraphs = dataclasses.field(repr=False)
+    paragraphs: List[documentai.Document.Page.Paragraph]
+    dimensions = dataclasses.field(repr=False)
+    dimensions: documentai.Document.Page.Dimension
+    document_text = dataclasses.field(repr=False)
+    document_text: str
+    documentai_block = dataclasses.field(repr=False)
+    documentai_block: documentai.Document.Page.Block
+    id = dataclasses.field(repr=False)
+    id: str
+    hocr_paragraphs = dataclasses.field(init=False, repr=False)
+    hocr_paragraphs: List[hOCR_Paragraph]
+    bounding_box = dataclasses.field(init=False, repr=False)
+    bounding_box: str
+
+    def __post_init__(self):
+        self.hocr_paragraphs = _load_hocr_paragraphs(paragraphs=(self.paragraphs), page=(self.page), document_text=(self.document_text))
+        self.bounding_box = _get_bounding_box(object=(self.documentai_block), dimensions=(self.dimensions))
+
+
+@dataclasses.dataclass
+class hOCR_Page:
+    documentai_page = dataclasses.field(repr=False)
+    documentai_page: documentai.Document.Page
+    document_text = dataclasses.field(repr=False)
+    document_text: str
+    hocr_blocks = dataclasses.field(init=False, repr=False)
+    hocr_blocks: List[hOCR_Block]
+    bounding_box = dataclasses.field(init=False, repr=False)
+    bounding_box: str
+    title = dataclasses.field(init=False, repr=False)
+    title: str
+    id = dataclasses.field(init=False, repr=False)
+    id: str
+
+    def __post_init__(self):
+        self.hocr_blocks = _load_hocr_blocks(page=(self.documentai_page), document_text=(self.document_text))
+        self.bounding_box = _get_bounding_box(object=(self.documentai_page), dimensions=(self.documentai_page.dimension))
+        self.title = 'Document'
+
+
+@dataclasses.dataclass
+class hOCR:
+    documentai_pages: List[documentai.Document.Page] = dataclasses.field(repr=False)
+    documentai_text: List[documentai.Document.Page] = dataclasses.field(repr=False)
+    filename: str = dataclasses.field(repr=False)
+
+    hocr_pages: List[hOCR_Page] = dataclasses.field(init=False, repr=False)
+    metadata: str = dataclasses.field(init=False, repr=False)
+
+    def __post_init__(self):
+        self.hocr_pages = _load_pages(pages=(self.documentai_pages), document_text=(self.documentai_text))
+
+    def export_hocr(cls):
```

Comment:
Why `cls`?  If this is meant to be classmethod, decorate it as so.

There is also no good way to unit test parts of this function as written.

Reply:
You could mock the respective objects and assert the returned string, and you can then test the respective objects _load functions.